src/pages/admin/donate/components/generic-details/GenericDetails.test.tsx

```
@@ -0,0 +1,165 @@
+        expect(screen.queryByText('No Items')).not.toBeInTheDocument();
+    });
+
+    it('removes item on delete', () => {
+        render(<GenericDetails {...defaultProps} />);
+        expect(screen.getByText('Item 1')).toBeInTheDocument();
+        fireEvent.click(screen.getByText('Delete'));
+        expect(screen.queryByText('Item 1')).not.toBeInTheDocument();
+    });
+
+    it('does not render children when collapsed', () => {
+        render(<GenericDetails {...defaultProps} children={({ formState }) => <span>{formState.name}-child</span>} />);
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)